man/systemd.service.xml

```diff
@@ -756,14 +756,19 @@
         If set to <option>no</option> (the default), the service will
         not be restarted. If set to <option>on-success</option>, it
         will be restarted only when the service process exits cleanly.
-        In this context, a clean exit means an exit code of 0, or one
-        of the signals
-        <constant>SIGHUP</constant>,
-        <constant>SIGINT</constant>,
-        <constant>SIGTERM</constant> or
-        <constant>SIGPIPE</constant>, and
-        additionally, exit statuses and signals specified in
-        <varname>SuccessExitStatus=</varname>. If set to
+        In this context, a clean exit means any of the following:
+        <itemizedlist>
+            <listitem><simpara>Exit code of 0.</simpara></listitem>
+            <listitem><simpara>One of the signals
+                <constant>SIGHUP</constant>,
+                <constant>SIGINT</constant>,
+                <constant>SIGTERM</constant> or
+                <constant>SIGPIPE</constant> (except for
+                <varname>Type=oneshot</varname>).</simpara></listitem>
```

Comment:
Oxford comma please.
It is not clear if the added parenthetical remark applies to SIGPIPE or all. Actually I assumed the former, and had to look at the code to understand this. Maybe reword this as
> For types other than Type=oneshot, one of the signals ...

Also, please use ";" instead of "." at the end of the items.

Reply:
Updated.